.github/workflows/generate_and_push_deriv_api_types.yml

```diff
@@ -24,7 +24,7 @@ jobs:
         run: |
           npm i json-schema-to-typescript
           npm i @deriv/api-types@latest
-          echo ::set-output name=old_declarations_hash::"$(git hash-object ./node_modules/@deriv/api-types/index.d.ts)"
+          echo name=old_declarations_hash::"$(git hash-object ./node_modules/@deriv/api-types/index.d.ts)" >> $GITHUB_OUTPUT
```

Comment:
`echo old_declarations_hash="$(git hash-object ./node_modules/@deriv/api-types/index.d.ts)" >> $GITHUB_OUTPUT`

I think it should be like this ? {name}={value}

I saw this example for the updated version here in github blog

<img width="722" alt="image" src="https://github.com/binary-com/deriv-app/assets/129206554/00b36cd6-3612-4523-a0cf-12ebd1712f5f">

Reply:
https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

Reply:
i also saw it, but didn't use it because we were not using the mentioned approach even before.
Honestly not sure how it works!
i will check the implemented one and if it didn't work i will proceed to use your mentioned approach